cmake/ConfigureSummary.cmake

```
@@ -188,6 +188,9 @@ function( NEST_PRINT_CONFIG_SUMMARY )
   message( "The NEST executable will be installed to:" )
   message( "  ${CMAKE_INSTALL_FULL_BINDIR}/" )
   message( "" )
+  message( "Shared objects (.so files) will be installed to:" )
```

```suggestion
  message( "NEST dynamic libraries and user modules will be installed to:" )
```
The rationale is that "shared objects" is a pretty low-level technical term and the files end in `.dylib` on macOS and `.dll` on Windows.

---

Good point!